public/index.html

```diff
@@ -4275,7 +4275,7 @@ <h4 data-i18n="STscript Settings">STscript Settings</h4>
                                             <span class="fa-solid fa-circle-question note-link-span"></span>
                                         </a>
                                     </label>
-                                    <label class="checkbox_label" title="Prevents {{getvar::}} {{getglobalvar::}} macros from having literal macro-like values auto-evaluated.&NewLine;e.g. &quot;{{newline}}&quot; remains as literal string &quot;{{newline}}&quot;&NewLine;&NewLine;(This is done by internally replacing {{getvar::}} {{getglobalvar::}} macros with scoped variables.)" data-i18n="[title]Prevents {{getvar::}} {{getglobalvar::}} macros from having literal macro-like values auto-evaluated.&NewLine;e.g. &quot;{{newline}}&quot; remains as literal string &quot;{{newline}}&quot;&NewLine;&NewLine;(This is done by internally replacing {{getvar::}} {{getglobalvar::}} macros with scoped variables.)">
+                                    <label class="checkbox_label" title="Prevents {{getvar::}} {{getglobalvar::}} macros from having literal macro-like values auto-evaluated.&NewLine;e.g. &quot;{{newline}}&quot; remains as literal string &quot;{{newline}}&quot;&NewLine;&NewLine;(This is done by internally replacing {{getvar::}} {{getglobalvar::}} macros with scoped variables.)" data-i18n="[title]Prevents {{getvar::}} {{getglobalvar::}} macros from having literal macro-like values auto-evaluated. e.g. &quot;{{newline}}&quot; remains as literal string &quot;{{newline}}&quot;  (This is done by internally replacing {{getvar::}} {{getglobalvar::}} macros with scoped variables.)">
```

Comment:
If you change the i18n string to be different from the exact content of the value, you likely should use a shortened key here. Like we already do in many other places. `stscript_parser_flag_replace_getvar_label` or something.

Reply:
fixed in https://github.com/SillyTavern/SillyTavern/commit/3e5f98223d0c45ea1a94f793bcc075c84ef470cd